Documentation/tutorial/intro markdown lite.md

```diff
@@ -0,0 +1,183 @@
+# Markdown Lite
+
+## Introduction
+
+Markdown lite is a simple markdown tool to markup `md` file.
+
+## Design goal
+
+We write this tool for good extensiblity, so our implements should obay following order:
+
+1.  Extensiblity:
+    * Support markdown syntax extension.
+    * Support validation extension.
+2.  Correctness:
+    We follow GFM syntax, but when some rule is too hard to implement, just breaking.
+3.  Performance:
+    We almost don't care performance.
```

Comment:
How about another saying "Performance is not out major concern" or some better expression...?